src/main/java/Deadlines.java

```diff
@@ -0,0 +1,23 @@
+public class Deadlines extends Task {
+    private String endTime;
+    private String taskLabel = "[D]";
+    public Deadlines (String input){
+        super(input.substring(9,input.indexOf('/') - 1)); // Sanitize input by removing "deadline" at the start
+        super.setTaskLabel(taskLabel);
+        endTime = "(" + getEndTime(input) + ")";
+    }
+    // Method of StringBuffer operation taken from
```

Comment:
I like the way you use `StringBuffer` to operate on these strings, it's really useful.